libraries/botbuilder/tests/cloudAdapter.test.js

```diff
@@ -87,6 +87,29 @@ describe('CloudAdapter', function () {
 
             mock.verify();
         });
+
+        it('calls processActivityDirect', async function () {
+            const logic = async (context) => {
+                context.turnState.set(INVOKE_RESPONSE_KEY, {
+                    type: ActivityTypes.InvokeResponse,
+                    value: {
+                        status: 200,
+                        body: 'invokeResponse',
+                    },
+                });
+            };
+
+            const mock = sandbox.mock(adapter);
+            mock.expects('processActivity')
+                .withArgs('Bearer Authorization', { type: 'invoke', value: 'invoke' }, logic)
+                .once()
+                .resolves();
+            mock.expects('connect').never();
+
+            await adapter.processActivityDirect('Bearer Authorization', { type: 'invoke', value: 'invoke' }, logic);
+
+            mock.verify();
+        });
```

Comment:
Once you review the try-catch block in the method, you should probably add another test to cover the error case.